infra/openshift-manifests/inference-service/service.yaml

```diff
@@ -7,6 +7,12 @@ metadata:
     app: inference-service
 spec:
   template:
+    metadata:
+      annotations:
+        # Target 10 requests in-flight per pod.
+        autoscaling.knative.dev/target: "1"
+        autoscaling.knative.dev/min-scale: "10"
+        autoscaling.knative.dev/max-scale: "50"
```

Comment:
As the inference service is completely CPU bound, I wanted to have a target=1, which means, one pod will only handle 1 request.